R/servers.R

```diff
@@ -125,11 +119,6 @@ discoverServers <- function(quiet = FALSE) {
 
 getDefaultServer <- function(local = FALSE, prompt = TRUE) {
    existing <- servers(local)
-   # if there are no existing servers, silently try to discover one to work with
```

Comment:
observation: In looking at the callers to `getDefaultServer`, I noticed that `connectUser` and `connectApiUser` have quite a bit of shared code, including the calls which err when no server target is found. Some of that shared logic could move into this function.

Reply:
Note; I'll reduce that duplication in a future PR.